tools/isobuild/bundler.js

```diff
@@ -592,6 +593,15 @@ class File {
     this._contents = options.data || null; // contents, if known, as a Buffer
     this._hashOfContents = options.hash || null;
     this._hash = null;
```

Comment:
Isn't there an option where we can just update the existing `sha1` hash to be a `sha256` hash instead of calculating two separate hashes?